iroh-api/src/p2p.rs

```diff
@@ -34,7 +34,9 @@ pub trait P2p: Sync {
 }
 
 #[async_trait]
-impl<'a> P2p for ClientP2p<'a> {
+impl P2p for ClientP2p {
+    /// XXX really should be an API that intos a peer id, and then also accepts
```

Comment:
Can we use this format for todos:
```rust
// TODO(faassen): should be an API that...
```

That way we can all be uniform & know who to ask for clarity if someone else is working on implementing that todo.

I'm going to add this preference to the contributor/onboarding documentation

Reply:
You don't need to change it for this PR, just going forward.

Reply:
:+1: